### DISCUSSION TITLE
HEL class columns not present in mapunit table?

### DISCUSSION THREAD

**Discussion by kevinwolz** [ncss-tech/soilDB#321] [Q&A] [answered]
No matter what mukeys I try, I only get `NA` back for `muhelcl`, `muwathelcl`, and `muwndhelcl` from the mapunit table. These columns seem empty in both SDA and the downloadable SSURGO CSV tables. Here's an example. Am I missing something?

```
query <- paste("
   SELECT mukey, muhelcl, muwathelcl, muwndhelcl
   FROM mapunit 
   WHERE mukey IN ('423294')")

SDA_query(query)
```

**Comment by brownag** [comment, marked as answer]
@kevinwolz 

The same named HEL columns are marked as "obsolete" in the NASIS "datamapunit" table/model but are still present (all empty) in "mapunit" in SSURGO/SDA. 

I suspect the change was to do with legal responsibilities and a consistent source of data for preliminary HEL determinations. 

Highly Erodible Land determinations are initially based on a "HEL Soils List" in effect January 1, 1990 for each soil survey. The SSURGO product in many cases has been at least partially updated since then. I would say--not knowing the exact reason these columns were made obsolete--that there is generally a concern that a modern product and a historical, legally-binding concept from 1990 might differ.

With update efforts across the country harmonizing mapunits across political boundaries, it could become a bit of a liability to report HEL classes that could differ from specifics of that 1990 list, or the corresponding spatial extents. Even if nothing changes with HEL the latest SSURGO data cannot be "authoritative" at this time. Legally the determinations have to come from the 1990 list and maps. Since planners must use the frozen data for HEL a solution seems to have been to eliminate the determination from the "non-authoritative" product (SSURGO).

If you are generally interested in data elements for assessing potential risk for erosion... I might suggest avoiding the concept of "HEL" and instead use other erosion interpretations or the numeric values of e.g. K factor (Kf, Kw), slope gradient etc. that would be considered in a HEL determination. 

If you are specifically interested in HEL determinations, I am not sure that we have a standard way of delivering this info as a part of SSURGO, I would need to investigate further. Even if you _could_ get a list based on old mapunit names/survey areas you would have the issue of relating that to modern mapunit polygons which could be different shape, and may not match live MUKEYs. 

Whether we can/How to access HEL determinations would be a good question for the [Soils Hotline](https://www.nrcs.usda.gov/resources/education-and-teaching-materials/national-soil-information-system-nasis) SoilsHotline@usda.gov. If you ask for more info from them please report back here or let me know via email!

**Reply by kevinwolz** [reply]
Thanks for the quick and detailed reply! This is absolutely fascinating to learn about. For my application, I don't need the legal certainty, so I I think I will just stick to Kw for water and WEI for wind.
